### PR TITLE
test: fix implicit fallthrough

### PR DESCRIPTION
**Pull request by vbatts**
Signed-off-by: Vincent Batts <vbatts@hashbangbash.com>

### CODE REVIEW

File: test/test_common/environment.cc [hunk 1]
```diff
@@ -126,8 +126,10 @@ std::string TestEnvironment::substitute(const std::string& str,
   switch (version) {
   case Network::Address::IpVersion::v4:
     out_json_string = std::regex_replace(out_json_string, lookup_family_regex, "v4_only");
+    FALLTHRU;
```
Comment (mattklein123) [inline test/test_common/environment.cc]: I don't think this actually makes sense given the context? Should probably be break here and below?

Comment (htuch) [inline test/test_common/environment.cc]: Yes, good point, please fix.

Comment (vbatts) [inline test/test_common/environment.cc]: right on!

Comment (vbatts) [inline test/test_common/environment.cc]: /me wasn't paying attention to the case, but just that it was implicitly falling through...

**Comment by vbatts** [comment]
updated

**Pull request merged**